pnpm-workspace.yaml

```diff
@@ -1,4 +1,5 @@
 packages:
-  - 'apps/*'
-  - 'sdks/*'
-  - '!sdks/java'
\ No newline at end of file
+  - "apps/*"
+  - "sdks/*"
+  - "!sdks/java"
+  - "integrations/*"
```

Comment:
Since `integrations/` folder would provide integration examples it might be better to exclude it from the workspaces to avoid unnecessary builds

Reply:
My interpretation from Francisco was that we should include in the project workspace the folders of the demo, and also to be have access to package to avoid the local reference.

Reply:
I can change the name for demos.

Reply:
changed.